### PR TITLE
Tools & Resources

### PR DESCRIPTION
**Pull request by ignapas**
# Description

Takes care of [this ticket](https://www.wrike.com/workspace.htm?acc=3203588#/task-view?id=676984160).

* Layouting for the new page, coming from [here](https://app.abstract.com/share/d7ffee59-e165-4809-a6c4-86c8cfc9c949?collectionLayerId=1afcd54d-36d3-4d5d-a6b0-926516897274&mode=design&sha=7bfba68b18f275d079c5e97f10c95556dd78fa64). Although some differences from the wireframe are kept for component reusability, a design review is very welcome.
* New content type created in Contentful for this layout. This page instance can be found [here](https://app.contentful.com/spaces/6bya4tyw8399/entries/1Yy2BEB0df8HxLNx2Ivsct).

toolsresources

**OTHER**
* Decouopled component Paper that is also used by the About page.
Screenshot from 2021-11-12 15-10-25
* Decoupled component Chip that is also used by News & Events page.
Screenshot from 2021-11-12 15-14-23

## Type of change

Delete those that don't apply.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Only manual testing.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/Chip/Chip.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
```
Comment (egauzens) [inline components/Chip/Chip.vue:1]: Sorry I forgot to bring this up the other day, but there is already a component for this Called SparcPill. Could you remove this component and use that one instead? It looks to be identical. It looks like we are using it for the datasets results, I must have forgot to use it when making the about page refactor

File: components/Paper/Paper.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    <img
+      v-if="imgSrc"
+      slot="image"
+      class="portal-image midnightblue-background"
+      :src="imgSrc"
+    />
+  </div>
+</template>
+
+<style lang="scss" scoped>
```
Comment (egauzens) [inline components/Paper/Paper.vue:34]: Please move the css style definition below the script

File: components/Resources/FeaturedResource.vue [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      <a v-if="buttonLink !== null" :href="buttonLink">
+        <el-button class="pink-button">
+          View resource
+        </el-button>
+      </a>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
```
Comment (egauzens) [inline components/Resources/FeaturedResource.vue:32]: Same here, please move the css style definition below the script

**Review by egauzens — CHANGES_REQUESTED** [review]
Nice work! Everything looks good, I like the creation of the Paper component. Just a couple minor things to address before submitting.

-Remove the Chip component and use SparcPill wherever the Chip is used
-move style definitions below script definition

**Comment by ignapas** [comment]
> Nice work! Everything looks good, I like the creation of the Paper component. Just a couple minor things to address before submitting.
> 
> -Remove the Chip component and use SparcPill wherever the Chip is used -move style definitions below script definition

Got it, thanks for the review. I will make the changes asap.

**Review by egauzens — APPROVED** [review]
Lgtm! Thanks for making those changes

**Pull request merged**